AntMe/Properties/Resources.Designer.cs

```diff
@@ -1,7 +1,7 @@
 ﻿//------------------------------------------------------------------------------
 // <auto-generated>
 //     This code was generated by a tool.
-//     Runtime Version:4.0.30319.34014
```

Comment:
Wäre jetzt auch nicht nötig gewesen die im Commit zu haben

Reply:
stimmt, soll ichs wieder aus machen?

Reply:
ich würde diese Fiels(Settings.designer.cs und Resources.Designer.cs) mla mit in die .gitignor aufnehemn, wäre das eine Lösung für die Zukunft?

Reply:
ach quatsch. Ist ja kein wirkliches Problem. Lass drin